crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -3,7 +3,8 @@ mod parallel_scope;
 use core::{marker::PhantomData, panic::Location};
 
 use super::{
-    Deferred, IntoObserverSystem, IntoSystem, RegisterSystem, Resource, RunSystemCachedWith,
+    Deferred, IntoObserverSystem, IntoSystem, RegisterSystem, RemoveSystem, Resource,
```

Comment:
```suggestion
    Deferred, IntoObserverSystem, IntoSystem, RegisterSystem, UnregisterSystem, Resource,
```

Reply:
This will fail formatting because it's not alphabetized :)